src/main.rs

```diff
@@ -36,6 +36,10 @@ enum Opt {
         /// Number of seconds for user to release keys on startup
         #[arg(short, long, default_value = "2")]
         delay: f64,
+
+        /// Wait forever for the device to be found
+        #[arg(short, long, action)]
+        wait: bool,
```

Comment:
`wait` on its own is a bit too vague IMO, it should describe what is being waited for/upon.
I'm also not generally a fan of short options for similar reasons. (I know that delay has a short form above, but it's not something I generally do!)

```suggestion
        #[arg(long, action)]
        wait_for_device: bool,
```